bitbadges/chain.json

```diff
@@ -0,0 +1,114 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "bitbadges",
+  "status": "live",
+  "chain_type": "cosmos",
+  "network_type": "mainnet",
+  "website": "https://bitbadges.io/",
+  "pretty_name": "BitBadges",
+  "chain_id": "bitbadges-1",
+  "bech32_prefix": "cosmos",
+  "daemon_name": "bitbadgeschaind",
+  "node_home": "$HOME/.bitbadgeschaind",
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ubadge",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.00025,
+        "average_gas_price": 0.0025,
+        "high_gas_price": 0.025
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "ustake"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/bitbadges/bitbadgeschain/",
+    "recommended_version": "v1.0-mainnet",
+    "compatible_versions": [
+      "v1.0-mainnet"
+    ],
+    "versions": [{
+      "name": "v1.0-mainnet",
+      "binaries": {
+        "linux/amd64": "https://github.com/BitBadges/bitbadgeschain/releases/download/v1.0-mainnet/bitbadgeschain-linux-amd64",
+        "linux/arm64": "https://github.com/BitBadges/bitbadgeschain/releases/download/v1.0-mainnet/bitbadgeschain-linux-arm64"
+      },
+      "cosmos_sdk_version": "v0.50.8",
```

Comment:
the "sdk" and "ibc" properties are also required. They describe the same version data.